src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/TableOfContentPageObject.java

```diff
@@ -54,16 +57,12 @@ public void clickOnTOCListElement(int index) {
     listOfLinks.get(index).click();
   }
 
-  public boolean isUserMovedToRightSection(int index) {
-    JavascriptExecutor js = (JavascriptExecutor) driver;
-    String h2PosString =
-        js.executeScript(
-            "return Math.floor($('section.article-body h2').eq(" + index + ").offset().top)")
-            .toString();
-    int h2Pos = Integer.parseInt(h2PosString);
-    int windowYPos =
-        Integer.parseInt(js.executeScript("return $(window).scrollTop()").toString());
-    return h2Pos == windowYPos;
+  public boolean isUserMovedToSectionByIndex(int sectionIndex) {
+    String sectionIndexString = String.valueOf(sectionIndex);
+    By sectionBy = By.cssSelector(section_selector.replace("__index__", sectionIndexString));
+    WebElement section = driver.findElement(sectionBy);
```

Comment:
I will use this alternative:
WebElement section = driver.findElement(By.cssSelector(section_selector.replace("__index__", sectionIndex)));

I can't use the second alternative, the css selector should be part of UI mapping (which is on top of the page component)